src/pc/prometheus.rs

```diff
@@ -68,6 +68,8 @@ struct MetricsInner {
 	bitfields: IntGaugeVec,
 	/// Average candidate inclusion time measured in relay chain blocks.
 	para_block_times: HistogramVec,
+	/// Finality lag
+	finality_lag: IntGaugeVec,
```

Comment:
Doesn't need to be `GaugeVec`. The finality lag is for the relay chain.

Reply:
Switched to Gauge